<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/TrabajoClaseestadistica/blob/main/Actividad_1_estadistica2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

YEISON DAVID LEON HERNADEZ

In [2]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom

En una elección, un candidato tiene una probabilidad del $60\%$ de recibir un voto en cada votación individual. Si se emiten 20 votos.

1.   ¿Cuál es la probabilidad de que al menos obtenga 12 votos?
2.   ¿Cuál es la probabilidad de que lo más obtenga 5 votos?
3.  ¿Cuál es la probabilidd de que no obtenga votos.
4.  ¿Cuál es la probabilidad de que obtenga entre 8 y 10 votos?


¿Cuál es la probabilidad de que al menos obtenga 12 votos?

In [6]:
n=20
p=0.6
x=12
b=1-binom.cdf(x-1,n,p)
print(b)
m=b*100
print('la probabilidad es',m,'%')

0.5955987253122248
la probabilidad es 59.55987253122248 %


 la probabilidad de que al menos obtenga 12 votos es del 59%


¿Cuál es la probabilidad de que a lo más obtenga 5 votos?


In [7]:
n=20
p=0.6
x=5
p=binom.cdf(x,n,p)
m=p*100
print(p)
print('la probabilidad es es',m,'%')

0.001611524643456288
la probabilidad es es 0.1611524643456288 %


 la probabilidad de que a lo más obtenga 5 votos es de 0.16%

¿Cuál es la probabilidd de que no obtenga votos?

In [8]:
n=20
p=0.6
x=0
p=binom.pmf(x,n,p)
m=p*100
print(p)
print('la probabilidad es es',m,'%')

1.0995116277760013e-08
la probabilidad es es 1.0995116277760013e-06 %


la probabilidd de que no obtenga votos es  1.0995116277760013e-06 %

¿Cuál es la probabilidad de que obtenga entre 8 y 10 votos?


In [11]:
n=20
p=0.6
x=10
p=binom.cdf(x,n,p)
m=p*100
print(p)
print('la probabilidad es es',m,'%')

0.24466279668360685
la probabilidad es es 24.466279668360684 %


In [12]:
n=20
p=0.6
x=8
p=binom.cdf(x,n,p)
m=p*100
print(p)
print('la probabilidad es es',m,'%')

0.056526367034253025
la probabilidad es es 5.652636703425302 %


In [18]:
(0.24466279668360685-0.056526367034253025)*100


18.813642964935383

 la probabilidad de que obtenga entre 8 y 10 votos es de 18.8%